src/ui_utils.c

```diff
@@ -2300,7 +2300,6 @@ static void ui_menu_move(GtkWidget *menu, GtkWidget *old, GtkWidget *new)
 	g_object_unref(menu);
 }
 
-
```

Comment:
Any reason for removing this line?

Reply:
Just me being stupid and not checking the code carefully enough before committing. I'll put it back.

Reply:
No worries, thanks for reverting it.